src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -256,7 +256,7 @@ class SelectorDropdown extends React.Component<Props, State> {
         }
 
         const list = (
-            <ul className="overlay" id={listboxID} role="listbox">
+            <ul className={`overlay ${overlayTitle ? overlayTitle.toLowerCase() : ''}`} id={listboxID} role="listbox">
```

Comment:
can you use `classNames` library here? also, where does overlayTitle come from?

Reply:
will use classNames. overlayTitle comes from PillSelectorDropdown as a prop, which is passed in from ContactSelector in EUA.